cpp/data/sql_updates/ub_tools.27

```diff
@@ -0,0 +1 @@
+ALTER TABLE delivered_marc_records MODIFY COLUMN delivery_state ENUM('automatic', 'manual', 'error', 'ignore', 'reset', 'online_first') DEFAULT 'automatic' NOT NULL;
```

Comment:
This change should also be added to data/installer/ub_tools.sql

Reply:
And => did you implement any logic to skip the download for a certain timespan? I guess we also need an extra column for this.

Reply:
As to the extra column: see comment below.